connect/src/com/telenor/connect/AbstractSdkProfile.java

```diff
@@ -166,6 +166,10 @@ protected void previewParameters(ParametersHolder parameters) {
                         && isEmpty(wellKnownConfig.getNetworkAuthenticationTargetUrls()))) {
                     parameters.add("login_hint", String.format("MCCMNC:%s", mccMnc));
         }
+
+        if (!ConnectSdk.isCellularDataNetworkConnected()) {
```

Comment:
Yes, if it works on your device, I'd say it's unpredictable then :/
Yes, I was testing with 67. You can see on Monday, with the white Android phone.
IIRC I saw some Stackflow posts describing that in the past, that this is an unreliable method for detecting connectivity.